.circleci/config.yml

```diff
@@ -834,6 +834,18 @@ jobs:
       <<: *infrastructure_config_environment
       PKG_TARGET_OS: ubuntu
     steps: *upload_latest_artifacts
+
+
+  verify_release_artifacts:
+    executor: infrastructure_container_stable
+    <<: *infrastructure_config
+    working_directory: /infrastructure
+    steps:
+      - run:
+          name: Verify release artifacts
+          command: |
+            ./scripts/check_release_artifacts.sh ${CIRCLE_TAG}
```

Comment:
I see https://github.com/aeternity/infrastructure/commit/4539fd4ac6a02fc95b5db140e36c231d41678c34

Reply:
I see Windows package not checked. And it makes sense, as upload still manual.